recipes/rb-csl-styles/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "csl-styles" %}
+{% set version = "1.0.1.9" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 5899db4404ef96e6d836ad2970f82c2c7e960dd6aa00e3feb58daff01720e882
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+    - rb-csl >=1.0,<2
+
+test:
+  commands:
+    - ruby -r {{ name|replace('-','/') }} -e 'exit 0'
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: CC-BY-SA-3.0
```

Comment:
This is the only strange package in the PR. It is licensed under the Creative Commons, but the actual license is not include  in the package or in the github repo. I also could not easily find a text only version of CC-BY-SA-3.0.

I don't think this is a reason to block merging this PR.